src/Apps/Artist/Routes/Overview/index.tsx

```diff
@@ -77,86 +77,93 @@ class OverviewRoute extends React.Component<OverviewRouteProps, State> {
-            {showSelectedExhibitions && (
-              <>
-                <SelectedExhibitions
-                  artistID={artist.id}
-                  totalExhibitions={this.props.artist.counts.partner_shows}
-                  exhibitions={this.props.artist.exhibition_highlights}
-                />
-                <Spacer mb={1} />
-              </>
-            )}
-            {showArtistBio && (
-              <>
-                <ArtistBio
-                  onReadMoreClicked={() => {
-                    this.setState({ isReadMoreExpanded: true })
-                  }}
-                  bio={artist}
-                />
-                <Spacer mb={1} />
-              </>
-            )}
-            <>
-              <Media at="xs">
-                {bioLen < MAX_CHARS.xs ? (
-                  <>
-                    <Genes artist={artist} />
-                    <Spacer mb={1} />
-                  </>
-                ) : (
-                  showGenes && <Genes artist={artist} />
-                )}
-              </Media>
-              <Media greaterThan="xs">
-                {bioLen < MAX_CHARS.default ? (
-                  <>
-                    <Genes artist={artist} />
-                    <Spacer mb={1} />
-                  </>
-                ) : (
-                  showGenes && <Genes artist={artist} />
-                )}
-              </Media>
-
-              {showGenes && <Spacer mb={1} />}
-            </>
-            {showConsignable && (
-              <>
-                <Spacer mb={2} />
-                <Sans size="2" color="black60">
-                  Want to sell a work by this artist?{" "}
-                  <a
-                    href="/consign"
-                    onClick={this.handleConsignClick.bind(this)}
-                  >
-                    Learn more
-                  </a>.
-                </Sans>
-              </>
-            )}
+            {showArtistInsightsV2 && showArtistBio && BioFragment}
+            {showArtistInsightsV2 && showGenes && GenesFragment}
+            {showArtistInsightsV2 &&
+              showConsignable && (
+                <>
+                  {ConsignmentFragment}
+                  <Spacer mb={1} />
+                </>
+              )}
+            {ArtistInsights && <ArtistInsights artist={artist} />}
+            {!showArtistInsightsV2 &&
+              showSelectedExhibitions && (
+                <>
+                  <SelectedExhibitions
+                    artistID={artist.id}
+                    totalExhibitions={this.props.artist.counts.partner_shows}
+                    exhibitions={this.props.artist.exhibition_highlights}
+                  />
+                  <Spacer mb={1} />
+                </>
+              )}
+            {!showArtistInsightsV2 && showArtistBio && BioFragment}
+            {!showArtistInsightsV2 && showGenes && GenesFragment}
+            {!showArtistInsightsV2 && showConsignable && ConsignmentFragment}
```

Comment:
This isn't a blocker, but would it be possible to to organize this around some kind of switch statement that duplicates markup so that a conditions aren't woven throughout? 

```js
if (showArtistInsightsV2) {
  return (
     ... 
  )
} else ... 
```

Totally understand if these conditions are best, but wondering if we might be trying to adhere to DRY unnecessarily.

Reply:
Yep, agreed that it's a bit hard to read as is. I've pushed a commit which attempts to simplify it a bit. :+1: